voting/src/general_meeting.rs

```diff
@@ -209,6 +210,17 @@ impl GeneralMeetingManager for Context {
         };
         Ok(meeting)
     }
+
+    fn get_vote_box(&mut self, key: &[u8]) -> Result<VoteBox, Error> {
+        if !self.storage().has(key) {
+            return Err(Error::MeetingNotFound)
```

Comment:
Shouldn't we use `VoteBoxNotFound` instead of `MeetingNotFound`?

```suggestion
            return Err(Error::VoteBoxNotFound)
```